Assets/Scripts/Utility/MacroHelper.cs

```diff
@@ -228,12 +228,22 @@ public static class MacroHelper
             { "%wth", Worth }, // Worth
         // DF Unity - new macros:
             { "%pg", PlayerPronoun },   // He/She (player)
-            { "%pg1", PlayerPronoun },  // His/Her (player)
+            { "%pg1", PlayerPronoun },  // He/She (player)
```

Comment:
Why this change? (and why in this unrelated PR)
`Assets/StreamingAssets/Quests/L0A01L00.txt
24:<ce>               gold coin for %pg1 first sanctioned death.
`

Reply:
> Why this change? (and why in this unrelated PR)

Look carefully, I only changed comments, nothing functional

Reply:
Even if they're just comments they're unrelated to this PR
And the only occurence of %pg1 I could find was introduced by @JayH2971  
```
commit 8e03008fc7bf6a5cba04ea419212934da8f68144
Author: JayH2971 <37541932+JayH2971@users.noreply.github.com>
Date:   Fri Jul 3 16:26:17 2020 -0700

    DB, Knight, MG, FG typos

diff --git a/Assets/StreamingAssets/Quests/L0A01L00.txt b/Assets/StreamingAssets/Quests/L0A01L00.txt
index 08744a069..b380f72b3 100644
--- a/Assets/StreamingAssets/Quests/L0A01L00.txt
+++ b/Assets/StreamingAssets/Quests/L0A01L00.txt
@@ -22,7 +21,7 @@ QuestComplete:  [1004]
 <ce>               town, you will know if there are any guild
 <ce>                  buildings there. It is customary for
 <ce>               the novice to receive payment of a single
-<ce>               gold coin for his first sanctioned death.
+<ce>               gold coin for %pg1 first sanctioned death.
 <ce>                             Here is yours.
 
 RumorsDuringQuest:  [1005]

```

So maybe a better fix would be to drop %pg1 handling altogether

Reply:
Please see my comments. It was a typo by me. Sorry. :(